### ISSUE TITLE
installing archlinux image crashes because mbusb-EFI-loader not found

### ISSUE THREAD

**Issue by azrdev** [mbusb/multibootusb#123]
When trying to install the archlinux ISO, mbusb crashes with the following backtrace:
~~~
...
Trying to create loopback.cfg
loopback.cfg file is set to /run/media/root/AZMULTIBOOT/multibootusb/archlinux-2017.03.01-dual/loopback.cfg
Unable to update entry in grub.cfg...
Updated entry in syslinux.cfg...
EFI image does not exist. Copying now...
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/scripts/mbusb_gui.py", line 270, in install_syslinux
    update_distro_cfg_files(config.iso_link, config.usb_disk, config.distro, config.persistence)
  File "/usr/lib/python3.6/site-packages/scripts/update_cfg_file.py", line 239, in update_distro_cfg_files
    os.path.join(config.usb_mount, 'EFI', 'BOOT'))
  File "/usr/lib/python3.6/shutil.py", line 251, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.6/shutil.py", line 114, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] Datei oder Verzeichnis nicht gefunden: '/usr/share/multibootusb/data/EFI/BOOT/bootx64.efi'
[1]    27569 abort      sudo multibootusb
~~~

There is no EFI directory in `/usr/share/multibootusb/data/`, I suspect this should be added to the installer.

**Comment by mbusb** [comment]
Did you install or run from source?
Do the following:
1. Uninstall old multibootusb
2. Download  latest source code (devel)
3. Run multibootusb from downloaded source directory

**Comment by azrdev** [comment]
I installed using the current devel HEAD, which should be b1e7a3ae1f5b97d2eda4b2eb0b4ee77220b66544, and the [original PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=multibootusb).

**Comment by mbusb** [comment]
Please do what I have mentioned in the previous comment. I hope it should work.

**Comment by alindt** [comment]
Tested source at current devel HEAD (a150156) successfully.
The problem might be with packaging and/or setup.py, install.py etc.

**Comment by alindt** [comment]
@azrdev, give the [latest release](https://github.com/mbusb/multibootusb/releases) (8.6.0) a try

**Comment by mbusb** [comment]
It works. You need to uninstall previous version and install latest one. Closing the issue.

**Comment by azrdev** [comment]
This packaging issue is fixed with release 8.8.0, it doesn't crash anymore
